alembic/versions/f92e90328dfd_create_temperature_table.py

```diff
@@ -0,0 +1,39 @@
+"""create temperature table
+
+Revision ID: f92e90328dfd
+Revises: f44762673e26
+Create Date: 2024-04-20 22:53:39.986232
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = 'f92e90328dfd'
+down_revision: Union[str, None] = 'f44762673e26'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('temperatures',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('city_id', sa.Integer(), nullable=False),
+    sa.Column('date_time', sa.DateTime(), nullable=False),
+    sa.Column('temperature', sa.Integer(), nullable=False),
+    sa.ForeignKeyConstraint(['city_id'], ['cities.id'], ),
```

Comment:
Ensure that the `cities` table and the `id` column exist and are correctly defined in your database schema, as this foreign key constraint depends on them.